script.js

```diff
@@ -1,56 +1,63 @@
-let bookList = [];
-
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
-
-function addBook(title, author) {
-  const book = new Book(title, author);
-  bookList.push(book);
-}
-
-function filterByTitle(item) {
-  return item.title !== 'title 1';
-}
-
-function addBookToStorage(title, author) {
-  window.localStorage.setItem(title, author);
-}
-
-function revomeFromStorage(title) {
-  window.localStorage.removeItem(title);
-}
-
-function removeBook(title, e) {
-  bookList = bookList.filter(filterByTitle);
-  e.parentNode.parentNode.removeChild(e.parentNode);
-  revomeFromStorage(title);
-}
-
-function display(title, author) {
-  const div = document.createElement('div');
-  div.innerHTML = `
-  <p>${title}</p>
-  <p>${author}</p>
-  <button onClick = "removeBook('${title}', this)">Delete</button>`;
-  const bookContainer = document.getElementById('bookContainer');
-  bookContainer.appendChild(div);
+class Books {
+  constructor() {
+    this.bookList = [];
+  }
+
+  addBook(title, author) {
+    const book = { title, author };
+    this.bookList.push(book);
+  }
+
+  static addBookToStorage(title, author) {
+    window.localStorage.setItem(title, author);
+  }
+
+  static display(title, author) {
+    const div = document.createElement('div');
+    div.innerHTML = `
+    <p>${title}</p>
+    <p>${author}</p>
+    <button class = "delete-buttons">Delete</button>`;
+    const bookContainer = document.getElementById('bookContainer');
+    bookContainer.appendChild(div);
+  }
+
+  removeBook(title, e) {
+    this.bookList = this.bookList.filter(this.filterByTitle);
+    e.parentNode.parentNode.removeChild(e.parentNode);
+    this.revomeFromStorage(title);
+  }
+
+  static filterByTitle(item) {
+    return item.title !== 'title 1';
+  }
+
+  static revomeFromStorage(title) {
+    window.localStorage.removeItem(title);
+  }
 }
 
 for (let i = 0; i < localStorage.length; i += 1) {
   const title = localStorage.key(i);
   const author = localStorage.getItem(localStorage.key(i));
-  display(title, author);
+  const book = new Books();
+  book.display(title, author);
```

Comment:
- [ ] **The book is not rendered in the DOM after being created. Please fix this.**

This is happening because you are using a static class method with an instance of that class. Class methods should be applied directly on the classes. If you wish to define methods for the instances of the class, don't use the `static` keyword.